src/helpers/formatters.js

```diff
@@ -12,11 +12,16 @@ export default {
      * @param value
      * @param args
      */
-    currency: (value, ...args) => {
-        let currencyFormatter = global.currency;
+    formatCurrency: (value, ...args) => {
+        let currencyFormatter = global.OSREC && global.OSREC.CurrencyFormatter;
+        let handlebars = global.Handlebars;
 
-        if (!currencyFormatter) {
-            currencyFormatter = require('currency-formatter');
+        if (isUndefined(currencyFormatter)) {
+            currencyFormatter = require('currencyformatter.js');
+        }
+
+        if (isUndefined(handlebars)) {
```

Comment:
I think we should do `handlebars = require('handlebars');` inside the `if` block there and should add it in the shimify config just like what we did for `moment` and other dependencies.